cpp/src/tsne/tsne.cu

```diff
@@ -96,6 +105,64 @@ void TSNE_fit(const cumlHandle &handle, const float *X, float *Y, const int n,
   auto d_alloc = handle.getDeviceAllocator();
   cudaStream_t stream = handle.getStream();
 
+  //---------------------------------------------------
+  // PCA Intialization via Divide n Conquer Eigendecomposition
+  float *A;
+  device_buffer<float> X_C_contiguous(d_alloc, stream);
+
+  if (pca_intialization == true) {
```

Comment:
Should just be `if(pca_initialization)` right?

Reply:
Yep :) Sometimes I forget you don't need == true. :)